### ISSUE TITLE
Is it possible for a WeMos D1 ?

### ISSUE THREAD

**Issue by RonXTCdaBass** [scottlawsonbc/audio-reactive-led-strip#372]
Hi and hello!
i´m searching to use audio-reactive with my WeMos but all i tried seems not working...

maybe helpful...
 Boot Vers: 31
 CPU: 80
 SDK: 2.1.0(deb1901)
 Chip ID: 8073493
 Flash ID: 1454108

**Comment by joeybab3** [comment]
Yes it's possible, that's the device I used for testing as well as a nodemcu
